### DISCUSSION TITLE
No FluentMigrator packages folders after installing packages

### DISCUSSION THREAD

**Discussion by Gengaeldelse1** [fluentmigrator/fluentmigrator#1718] [Q&A]
Hey all, I'm in need of some assistance.  

I'm trying to set up a database repo to use FluentMigrator for deployments and I'm running into an issue.  I'm following the the FluentMigrator documentation, but when I install the following NuGet packages, there are no folders for FluentMigrator in the packages folder.

FluentMigrator
FluentMigrator.Console
FluentMigrator.Runner
FluentMigrator.Runner.SQLite
Microsoft.Data.Sqlite

I'm specifically looking for the Migrate.exe that is supposed to be part of the FluentMigrator.Console package, but after grepping the entire solution, the exe is not found.

Any thoughts on what I'm missing or what may be going on?

Thanks in advance.

**Comment by fubar-coder** [comment]
The packages folder only exists if you're using `packages.config` instead of `PackageReference`. You can find more information about `PackageReference` in the [documentation about `PackageReference` in project files](https://learn.microsoft.com/en-us/nuget/consume-packages/package-references-in-project-files) and other pages in this section.

**Reply by Gengaeldelse1** [reply]
Thank you.  

I did manage to stumble upon the packages folder in my User folder, and found the files I needed.  I will read up on this though.
